### PR TITLE
Implement api

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
1) Created  new game with the name of own choice by using the API.
2) the ID of the game returned by API saved using localstorage.
3) Implemented the "Refresh" button that get all scores for the game created from the API 
4) Implemented the form "Submit" button that save a score for the game created.
5) Used arrow functions instead of the function keyword.
6) Used async and await JavaScript features to consume the API.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,11 +105,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"> Will be added later</a>
+## 🚀 Live Demo <a name="live-demo"> https://sajibarafatsiddiqui.github.io/project_leaderboard/dist/</a>
```
Comment (rotimiazeez) [inline README.md:108]: - The MIT license link in your README seems broken. Kindly fix it for a more professional README 😉.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @sajibarafatsiddiqui 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- App working as expected 👏🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
### Please ensure your commit message(s) always starts with a capital letter as it's considered best practice 👍🏽.
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by sajibarafatsiddiqui** [comment]
Hi @rotimiazeez ,

Thanks for noticing my commit messages. I changed the commit messages accordingly. Thanks

File: README.md (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply README.md:108]: The MIT license link has been added. Thanks

**Review by pinkmoon25 — APPROVED** [review]
Hi @sajibarafatsiddiqui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Git flow :heavy_check_mark: 
Readme :heavy_check_mark: 
Linters Check :heavy_check_mark: 
Good PR and Commit message :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**